sandbox.go

```diff
@@ -425,6 +428,78 @@ func (sb *sandbox) execFunc(f func()) {
 	sb.osSbox.InvokeFunc(f)
 }
 
+func (sb *sandbox) ResolveService(name string) ([]*net.SRV, []net.IP, error) {
+	hostinfo := false
+	srv := []*net.SRV{}
+	ip := []net.IP{}
+
+	log.Debugf("Service name To resolve: %v", name)
+
+	parts := strings.Split(name, ".")
+	if len(parts) < 3 {
+		return nil, nil, fmt.Errorf("invalid service name, %s", name)
+	}
+	if parts[0] == "__host_info__" {
```

Comment:
I don't think we should do any kind of special processing here for container/host demux. It should be encoded as part of the service name itself and added by the population code.